client/actions/import.js

```diff
@@ -83,20 +83,20 @@ export function importUsers(files, connectionId) {
 /*
  * Get the status of a job.
  */
-export function probeImportStatus() {
+export function probeImportStatus(jobId) {
   return (dispatch, getState) => {
     const reducer = getState().import;
     const currentJob = reducer.toJS().currentJob;
-    if (currentJob && currentJob.id) {
+    if ((currentJob && currentJob.id) || jobId) {
```

Comment:
for maintainability, would be better to pull `currentJobId` into it's own variable so we don't have the OR in so many places.

`const currentJobId =(currentJob && currentJob.id) || jobId`;